set.mm

```diff
@@ -318970,6 +318988,22 @@ can be the countable union of rare closed subsets (where rare means
     cmsss.h $e |- K = ( M |`s A ) $.
     cmsss.x $e |- X = ( Base ` M ) $.
     cmsss.j $e |- J = ( TopOpen ` M ) $.
+    $( The restriction of a metric space is closed if it is complete.
```

Comment:
`A subset of a metric space is closed if the metric space induced on it is complete.` ?
Maybe I'm nitpicking...